app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -288,6 +287,7 @@ def idv_result_to_form_response(
         end
       end
 
+      result[:biographical_info]&.delete(:same_address_as_id)
```

Comment:
Does this mutate the original result value in a way that might cause other side-effects? Is it something where we could either avoid assigning `same_address_as_id` into that result in the first place if it's not needed, or create a copy if this is being deleted to avoid including in `extra`?

e.g.

```rb
extra: extra.merge(
  proofing_results: {
    **result.except(:errors, :success),
    biographical_info: result[:biographical_info].except(:same_address_as_id),
  },
)
```

Reply:
@aduth See commit [20958fe](https://github.com/18F/identity-idp/pull/11400/commits/20958feaaa04088efa62646cd5b043f3f911c3fb).